README.md

```diff
@@ -1,27 +1,43 @@
 
 <h2>Please visit my wiki link for full list of questions</h2>
+
 <h3>https://github.com/mission-peace/interview/wiki</h3>
 
 <h2> Like my facebook page for latest updates on my youtube channel</h2>
 <h3>https://www.facebook.com/tusharroy25</h3>
 
 <h2> Contribution </h2>
-Please contribute to this repository to help it make better. Any change like new question, code improvement, doc improvement etc. is very welcome. Just send me a pull request and I will review the request and approve it if it looks good. 
+
+Please contribute to this repository to help it make better. Any change like new question, code improvement, docs improvement etc. is very welcome. Just send me a pull request and I will review the request and approve it if looks good. 
 
 <h2> How to use this repository </h2>
 
 <h3> Softwares to install </h3>
-* Install JDK8 https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html
-* Install Git https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
-* Install either Intellij https://www.jetbrains.com/idea/download/
-* If you like eclipse instead of intellij install eclipse https://eclipse.org/downloads/
+
+- Install JDK8 https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html
+
+- Install Git https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+- Install either Intellij https://www.jetbrains.com/idea/download/
+
+- If you like eclipse instead of intellij install eclipse https://eclipse.org/downloads/
```

Comment:
sorry for being nitpicky, but on line `21` you have used `intellij -- > Intellij`, I suggest using capitalized first letter for names of products, same thing applies for `eclipse --> Eclipse`